internal/configs/virtualserver.go

```diff
@@ -988,6 +992,72 @@ func (p *policiesCfg) addWAFConfig(
 	return res
 }
 
+func (p *policiesCfg) addBadosConfig(
+	Bados *conf_v1.Bados,
+	polKey string,
+	polNamespace string,
+	apResources map[string]string,
+) *validationResults {
+	res := newValidationResults()
+	if p.Bados != nil {
```

Comment:
I might be wrong but it looks like `p.Bados` is never reset to nil, so once it's set, it can never be changed?

It also looks like `addBadsConfig()` is called frequently, as it is downstream from `GenerateVirtualServerConfig`. Does that mean it will generate a warning every time a VirtualServer is added/changed ?